.github/workflows/pypi.yml

```diff
@@ -0,0 +1,32 @@
+name: "PyPI publishing"
+
+on:
+  release:
+    types: [published]
```

Comment:
note to @Jens-G : this means we only publish to pypi when we create a github release (not a git tag). please let me know if you'd prefer git tags instead.